cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -2053,6 +2053,9 @@ public static final class MetadataConsumer {
   public static final class SourceControlManagement {
     public static final String GIT_COMMAND_TIMEOUT_SECONDS = "source.control.git.command.timeout.seconds";
     public static final String GIT_REPOSITORIES_CLONE_DIRECTORY_PATH =
-      "source.control.git.repositories.clone.directory.path";
+        "source.control.git.repositories.clone.directory.path";
```

Comment:
we should be using consistent naming. `source.control.repository.clone.directory.path`